contracts/layer-2/ChampMarketplace.sol

```diff
@@ -367,24 +410,28 @@ contract ChampMarketplace is AccessControlEnumerableUpgradeable {
      *
      * Requirements:
      *
-     * - nMarketplacePercentFees must be a percentage (Between 0 and 100 included).
+     * - Sum of nMarketplaceSellPercentFees and nMarketplaceBurnPercentFees must be an integer between 0 and 100 included.
      * - Caller must have role FEE_MANAGER_ROLE.
      */
-    function setMarketplacePercentFees(uint256 nMarketplacePercentFees)
-        external
-        onlyRole(FEE_MANAGER_ROLE)
-    {
+    function setMarketplacePercentFees(
+        uint128 nMarketplaceSellPercentFees,
+        uint128 nMarketplaceBuyPercentFees,
+        uint256 nMarketplaceBurnPercentFees
+    ) external onlyRole(FEE_MANAGER_ROLE) {
         require(
-            nMarketplacePercentFees >= 0,
-            "ChampMarketplace: nMarketplacePercentFees should be positive"
+            nMarketplaceSellPercentFees + nMarketplaceBurnPercentFees <= 100,
```

Comment:
```suggestion
            nMarketplaceSellPercentFees + nMarketplaceBurnPercentFees + nMarketplaceBuyPercentFees <= 100,
```

Reply:
Just a reflection about this validation:

- `nMarketplaceSellPercentFees` + `nMarketplaceBurnPercentFees` must always be bellow 100% because these two fees are deducted directly from the listing price, so we can never deduct more than 100% of the price.
- `nMarketplaceBuyPercentFees` can be (in theory) more than 100%. We don't have any restrictions on this number, because this fee is added on top of the listing price. So, for a price of 100, we could theoretically have a 150% buyer fee which would mean the buyer would pay 250, fee receiver would receive 150 and seller would receive 100 (full price).
- `nMarketplaceSellPercentFees + nMarketplaceBurnPercentFees + nMarketplaceBuyPercentFees` this calculation assumes that the sum of ALL our fees needs to be bellow 100% which seems like a bit of an arbitrary restriction to impose and is not restriction defined for this feature.

Based on this, I'd leave the current implementation as is. WDYT?